pontoon/base/static/js/fluent_interface.js

```diff
@@ -317,7 +317,7 @@ var Pontoon = (function (my) {
           } else {
             var attributes = ast.attributes;
             if (attributes && attributes.length) {
-              response = attributes[0].value.elements[0].value;
+              response = this.serializePlaceables(attributes[0].value.elements);
```

Comment:
This fixes this bug:
https://pontoon.mozilla.org/sl/firefox-screenshots/server.ftl/?string=168505

Where the simplified version of the string is cut off if it contains placeables.